src/pages/Cart/index.js

```diff
@@ -64,24 +76,54 @@ function ProductCart() {
 }
 
 function Carting({ product }) {
+  const [quantity, setQuantity] = useState(1);
+  function addQuantity() {
```

Comment:
Please, use consistent function names. If the name of the function for reducing the number of items in the cart is `reduceQuantity`, then the name of the function for increasing the number of items should be `increaseQuantity`.

It makes your code more readable and easier to understand

Reply:
well understoood.